MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -370,10 +370,11 @@ public void FixedUpdate()
 				_timeSinceLastPhysicsUpdate += Time.fixedDeltaTime;
 
 				float updateDeltaTime = Math.Max(Time.fixedDeltaTime,
-					Time.fixedDeltaTime * (float)Math.Floor(_physicsUpdateTimestep / Time.fixedDeltaTime));
+					Time.fixedDeltaTime * (float)Math.Round(_physicsUpdateTimestep / Time.fixedDeltaTime));
```

Comment:
>Round [](start = 39, length = 5)

I actually like more if we keep Floor,
but update _physicsUpdateTimestep = 0.034f

Reply:
Why is floor better?

Reply:
I would round for the moment if you do not mind. should enable us to guess the right value better.

---
In reply to: [453654511](https://github.com/microsoft/mixed-reality-extension-unity/pull/218#discussion_r453654511) [](ancestors = 453654511)